src/general/OptionsMenu.cs

```diff
@@ -636,6 +653,46 @@ private void DisplayResolution()
             .FormatSafe(screenResolution.x, screenResolution.y);
     }
 
+    /// <summary>
+    /// Displays the GPU name, the display driver name and used video memory
+    /// </summary>
+    private void DisplayGpuInfo()
+    {
+        var videoAdapterName = VisualServer.GetVideoAdapterName();
+        gpuName.Text = TranslationServer.Translate("AUTO_GPU_NAME")
+            .FormatSafe(videoAdapterName);
+
+        if (OS.GetCurrentVideoDriver() == OS.VideoDriver.Gles2)
+        {
+            // Gles2 is being used
+            displayDriverName.Text = TranslationServer.Translate("CURRENT_DISPLAY_DRIVER_NAME")
+                .FormatSafe(TranslationServer.Translate("GLES2"));
+        }
+        else if (OS.GetCurrentVideoDriver() == OS.VideoDriver.Gles3)
+        {
+            // Gles3 is being used
+            displayDriverName.Text = TranslationServer.Translate("CURRENT_DISPLAY_DRIVER_NAME")
+                .FormatSafe(TranslationServer.Translate("GLES3"));
+        }
+        else
+        {
+            // An unknown display driver is being used
+            displayDriverName.Text = TranslationServer.Translate("CURRENT_DISPLAY_DRIVER_NAME")
+                .FormatSafe(TranslationServer.Translate("UNKNOWN_DISPLAY_DRIVER"));
+        }
+
+        float videoMemoryInMebibytes = VisualServer.GetRenderInfo(VisualServer.RenderInfo.VideoMemUsed);
+
+        // Convert to mebibytes
+        videoMemoryInMebibytes /= Constants.MEBIBYTE;
+
+        // Round to 2 places after the floating point
+        var decimalVideoMemory = Mathf.Round(videoMemoryInMebibytes * 100) / 100;
```

Comment:
Instead of doing this complex thing, you should just use `Math.Round((float)adasdsad, 2)` I didn't remember that Mathf round method didn't support specifying how many decimal places to use.